app-frontend/config/webpack/environments/development.js

```diff
@@ -21,11 +21,17 @@ module.exports = function (_path) {
             'history-api-fallback': true,
             port: port,
             proxy: {
-                '*': 'http://localhost:9100'

```

Comment:
We also haven't been really serving from the nginx container ... but I think it makes sense to do so because that's where we will catch some bugs around CORS, headers, etc.